crates/sui-replay/src/lib.rs

```diff
@@ -531,3 +538,11 @@ pub async fn execute_replay_command(
         }
     })
 }
+
+pub(crate) fn chain_from_chain_id(chain: &str) -> Chain {
+    match chain {
+        MAINNET_CHAIN_ID => Chain::Mainnet,
+        TESTNET_CHAIN_ID => Chain::Testnet,
+        _ => Chain::Unknown,
+    }
+}
```

Comment:
We also have this in core protocol: https://github.com/MystenLabs/sui/blob/c5ae3de27d17f6ae9b7f875f7b3e6966320dc706/crates/sui-types/src/digests.rs#L162-L172

Reply:
Sadly this wont work because the chain id from the rpc is only the first 4 bytes of the digest.
This is not enough to create the ChainIdentifier type.